src/subscription_manager/managercli.py

```diff
@@ -2891,6 +2891,11 @@ def _do_command(self):
                 print('- %s' % format_name(message, 2, columns))
             print('')
 
+        store = syspurposelib.SyncedStore
+        if store:
+            store = store(self.cp, consumer_uuid=self.identity.uuid)
```

Comment:
I see we're using this idiom in another place, but we have a method `get_sys_purpose_store` in the `syspurposelib` model.  Would it be appropriate to use that here instead?